tests/Feature/database/factories/UserFactory.php

```diff
@@ -8,6 +8,9 @@
     return [
         'name'  => $faker->name,
         'email' => $faker->email,
+        'should_reset_password' => rand(0, 1),
```

Comment:
Please don't use random in the factories. In another project, after using it for a while, it can lead to failures in x % of the time for certain tests.

It's probably not needed to be set here.

Reply:
Why? this value is always true or false so this should work. If a test is dependant on this value then it should be set inside the test and never be hidden inside factory

Reply:
Thus it can be removed from here.